test/src/Development/IDE/Test.hs

```diff
@@ -31,7 +31,10 @@ import Test.Tasty.HUnit
 type Cursor = (Int, Int)
 
 cursorPosition :: Cursor -> Position
-cursorPosition (line,  col) = Position line col
+cursorPosition (line,  col) =
+    Position
+        (fromInteger $ toInteger line)
```

Comment:
does `fromIntegral` work here?

Reply:
it does, thanks. However, since the test suite isn't buildable with ghc-lib, I simply undid this change